azure_functions_worker/bindings/meta.py

```diff
@@ -184,3 +215,57 @@ def to_outgoing_param_binding(binding: str, obj: typing.Any, *,
         return protos.ParameterBinding(
             name=out_name,
             data=rpc_val)
+
+
+def get_deferred_binding(bind_name: str,
+                         pytype: typing.Optional[type] = None) -> object:
+    binding = None
+
+    # Checks if pytype is a supported sdk type
+    if (DEFERRED_BINDINGS_REGISTRY is not None
+            and DEFERRED_BINDINGS_REGISTRY.check_supported_type(pytype)):
+        # Returns deferred binding converter
+        binding = DEFERRED_BINDINGS_REGISTRY.get(bind_name)
+
+    # This will return None if not a supported type
+    return binding
```

Comment:
```suggestion
    # This will return None if not a supported type
    return DEFERRED_BINDINGS_REGISTRY.get(bind_name) if (DEFERRED_BINDINGS_REGISTRY is not None
            and DEFERRED_BINDINGS_REGISTRY.check_supported_type(pytype)) else None
```